Modules/WRCore/Package.swift

```diff
@@ -0,0 +1,33 @@
+// swift-tools-version: 5.9
+
+import PackageDescription
+
+let package = Package(
+    name: "WRCore",
+    platforms: [
+        .iOS(.v15),
+    ],
+    products: [
+        .library(
+            name: "WRCore",
+            targets: ["WRCore"]
+        ),
+    ],
+    dependencies: [
+        .package(path: "../SharedUI"),
+        .package(url: "https://github.com/DatepollSystems/WaiterRobot-Shared-Android.git", from: "1.6.1"),
```

Comment:
Does this mean, that we have to define the version now twice (here and in the project.yml)?
The version specifications do not match (I think it would also be better to use here an exact version because shared might also have breaking changes in patch versions)

Best would of course be when we would not need to specify the version here again (or only here)

Reply:
As SwiftPackages and project.yml are different technologies - yes we need to define it twice.